elastic/integrations
# Make Vsphere GA: Enabling Vsphere Metrics/Logs

- Enhancement

## What does this PR do?
This PR enables below things in Vsphere:

- Receiving metrics/logs in integration
- Addition of system tests
- Update on dashboards
- Update on fields.yml
- Support for new metrices

## Checklist


## How to test this PR locally

Bring up the vcsim simulator on system and add integration. It should start the metrics flow to Kibana

## Related issues


- Closes https://github.com/elastic/integrations/issues/3508
